common/views/components/MainViewerPrototype/MainViewerPrototype.tsx

```diff
@@ -373,7 +374,7 @@ const MainViewer: FunctionComponent<Props> = ({
   }, [canvasIndex]);
 
   return (
-    <div data-test-id="main-viewer">
+    <div aria-live="assertive" data-test-id="main-viewer">
```

Comment:
Given that this is generally used for critical content changes, could you explain why `polite` wouldn't do?

Reply:
yeah, I'm not sure about this - but basically the search results links have a lot of text in them and if someone using a screen reader selects the link without 'assertive' on the main viewer, they continue to hear the link text being read out for quite sometime, before hearing the content in the main viewer - which seemed confusing to me, like the link didn't do anything.

Reply:
Fancy leaving a note here? Are we tracking questions we have for accessibility testing? @alicerichmond what's the easiest way for us to do this to make it useful to you?